control/controlclient/client.go

```diff
@@ -54,6 +54,10 @@ type Client interface {
 	// TODO: It might be better to simply shutdown the controlclient and
 	// make a new one when it's time to unpause.
 	SetPaused(bool)
+	// SetSleepMode pauses the control client and prevents anybody else
```

Comment:
"pauses"? Is that a copy/paste-o? The line immediately above is "SetPaused" so the reader will wonder how this is different from that SetPaused.